packages/core/src/base-axis-chart.ts

```diff
@@ -427,7 +445,7 @@ export class BaseAxisChart extends BaseChart {
 			if (secondaryYAxisRef.nodes().length > 0) {
 				secondaryYAxisRef.transition(t)
 					.attr("transform", `translate(${this.getChartSize().width}, 0)`)
-					// Being cast to any because d3 does not offer appropriate typings for the .call() function
+					// Being cast to any because d3 does noy-at offer appropriate typings for the .call() function
```

Comment:
`not` would be correct here
```suggestion
					// Being cast to any because d3 does not offer appropriate typings for the .call() function
```